clang/test/CIR/CodeGen/agg-init.cpp

```diff
@@ -1,7 +1,7 @@
 // RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -std=c++17 -fclangir-enable -Wno-unused-value -emit-cir %s -o %t.cir
 // RUN: FileCheck --input-file=%t.cir %s
 
-// CHECK: !ty_22yep_22 = !cir.struct<struct "yep_" {!u32i, !u32i}>
+// CHECK: !ty_22yep_22 = !cir.struct<struct "yep_" {!cir.int<u, 32>, !cir.int<u, 32>}>
```

Comment:
This means that we are gonna have printed CIR that uses `!u32i` because some operation will use it but a full `!cir.int<u, 32>` when it's a struct type. I'd prefer if we keep the good consistency.

Reply:
Unfortunately, mutable types do not support aliases yet, but there is a fix for this in review: https://github.com/llvm/llvm-project/pull/66663.

At some point, we will likely be able to use aliases on everything, so I suggest that, for now, we leave only the structs using full types and everything else using aliases.

Reply:
Tracked in #304